java/registry/src/main/java/io/opensaber/registry/service/DecryptionHelper.java

```diff
@@ -0,0 +1,36 @@
+package io.opensaber.registry.service;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.opensaber.registry.exception.EncryptionException;
+import io.opensaber.registry.util.Definition;
+import io.opensaber.registry.util.DefinitionsManager;
+import io.opensaber.registry.util.PrivateField;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.Map;
+
+@Component
+public class DecryptionHelper extends PrivateField {
+    @Autowired
+    private EncryptionService encryptionService;
```

Comment:
Move this Encryption service and DefinitionsManager to PrivateField. I see both Decryption/EncryptionHelper seems to need it.

Reply:
Yeah moved to PrivateField.